workflows/rnavar.nf

```diff
@@ -92,8 +93,10 @@ def seq_platform        = params.seq_platform ? params.seq_platform : []
 def seq_center          = params.seq_center ? params.seq_center : []
 
 // Initialize file channels based on params
-dbsnp                   = params.dbsnp             ? Channel.fromPath(params.dbsnp).collect()       : []
-dbsnp_tbi               = params.dbsnp_tbi         ? Channel.fromPath(params.dbsnp_tbi).collect()   : []
+dbsnp                   = params.dbsnp             ? Channel.fromPath(params.dbsnp).collect()               : Channel.empty()
+dbsnp_tbi               = params.dbsnp_tbi         ? Channel.fromPath(params.dbsnp_tbi).collect()           : Channel.empty()
+known_indels            = params.known_indels      ? Channel.fromPath(params.known_indels).collect()        : Channel.empty()
+known_indels_tbi        = params.known_indels_tbi  ? Channel.fromPath(params.known_indels_tbi).collect()    : Channel.empty()
```

Comment:
What about
```suggestion
dbsnp                   = params.dbsnp             ? Channel.fromPath(params.dbsnp).collect()               : []
dbsnp_tbi               = params.dbsnp_tbi         ? Channel.fromPath(params.dbsnp_tbi).collect()           : []
known_indels            = params.known_indels      ? Channel.fromPath(params.known_indels).collect()        : []
known_indels_tbi        = params.known_indels_tbi  ? Channel.fromPath(params.known_indels_tbi).collect()    : []
```
That should take care of lines +292 to +297

Reply:
if we change to [], then the concat() at line +246 to +247 will fail.
Earlier we used the following with [], but resulted in error (java.lang.NullPointerException: Cannot get property 'id' on null object) that Sarah faced when only known_indel file was provided without a dbSNP.

```
known_sites     = Channel.from([params.dbsnp, params.known_indels]).collect()
known_sites_tbi = Channel.from([params.dbsnp_tbi, params.known_indels_tbi]).collect()
```